packages/menu/src/Menu.js

```diff
@@ -50,12 +54,14 @@ export default class Menu extends Component {
   };
 
   static defaultProps = {
-    multiple: false
+    multiple: false,
+    unselect: true
```

Comment:
The `unselect` prop needs to be added to the `MenuGroup` component as well. You can see in the second `storybook` story that you can no longer unselect selected options. You may need to add more logic in the behaviors to make sure this acts like the `Menu`.